### DISCUSSION TITLE
Klipper profile for my Creality cr-X Pro

### DISCUSSION THREAD

**Discussion by alfa2960** [CrealityOfficial/Creality_Sonic_Pad#13] [Q&A]
Can someone give me a printer config or help me make it? I can’t. If you need any more information just ask. Thanks in advance.

**Comment by kamoteshake** [comment]
I hope [this](https://www.reddit.com/r/3Dprinting/comments/ci6cdb/finally_running_klipper_on_a_creality_crx_hope/) helps.
